ports/templates/ports/port-detail/summary.html

```diff
@@ -0,0 +1,89 @@
+{% load format_names %}
+    <br>
+    <div class="row">
+        <div class="col-lg-8">
+            <table class="table table-striped border border-top border-muted">
+                <tr>
+                    <th scope="row">Homepage</th>
+                    <td><a href="{{ port.homepage }}" target="_blank">{{ port.homepage }}</a></td>
+                </tr>
+                <tr>
+                    <th scope="row">Version:</th>
+                    <td>{{ port.version }}</td>
+                </tr>
+                <tr>
+                    <th scope="row">Maintained By:</th>
+                    <td>{% if maintainers.all|length > 0 %}
```

Comment:
probably move the `if` condition to below `<td>` to keep the if-else-endif in the same line.